### PR TITLE
Migrate runtime views to new perspective layout

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
Depends on https://github.com/kiegroup/appformer/pull/175

**Comment by cristianonicolai** [comment]
@nmirasch @bsremac please have a look.

**Comment by bsremac** [comment]
@cristianonicolai I have been having problems building _jbpm-wb_ since last Friday. The failure happens while compiling the last module - _jbpm-wb-showcase_. This is just one of many errors I see in the console:
```
[ERROR] Errors in 'jar:file:/home/bsremac/.m2/repository/org/kie/workbench/forms/kie-wb-common-forms-editor-client/7.7.0-SNAPSHOT/kie-wb-common-forms-editor-client-7.7.0-SNAPSHOT-sources.jar!/org/kie/workbench/common/forms/editor/client/editor/FormEditorPresenter.java'
[INFO]          [ERROR] Line 275: versionRecordManager cannot be resolved
[INFO]          [ERROR] Line 436: versionRecordManager cannot be resolved
[INFO]          [ERROR] Line 450: notification cannot be resolved
[INFO]          [ERROR] Line 285: The method isDirty(int) is undefined for the type FormEditorPresenter
[INFO]          [ERROR] Line 254: menus cannot be resolved to a variable
...
```
I don't know how to get pass this problem. I have gone as far as cleaning my entire local maven repo and then again pulling all dependencies and building appformer locally due to dependency on kiegroup/appformer#175. 
I can successfully build _kie-wb-common_ project, but even with all of this in place, I'm still getting the errors.
All of this made me think that certain group of .jars we depend on is really corrupted, but then you would have the same problem too and you obviously don't.

**Comment by cristianonicolai** [comment]
@bsremac I got the same issue this morning. I had to rebase my changes in https://github.com/kiegroup/appformer/pull/175 to make it work again. You will have to checkout the latest version from that PR locally and rebuild for it to work.

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/details/multi/advance/AdvancedProcessDefDetailsMultiPresenter.java [hunk 1]
```diff
@@ -56,27 +57,18 @@ public Position getPosition() {
...
-                            public IsWidget build() {
-                                return view.getOptionsButton();
-                            }
-                        };
-                    }
-                }).endMenu()
-
                 .newTopLevelCustomMenu(new RefreshMenuBuilder(this)).endMenu()
+                .newTopLevelCustomMenu(newInstanceMenu).endMenu()
+                .newTopLevelMenu(Constants.INSTANCE.Options())
```
Comment (bsremac) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/details/multi/advance/AdvancedProcessDefDetailsMultiPresenter.java:62]: Just out of curiosity, where does the magic happen which turns this into KebabMenu?

**Review by bsremac — APPROVED** [review]

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/details/multi/advance/AdvancedProcessDefDetailsMultiPresenter.java (hunk 1 above)
Comment (cristianonicolai) [reply jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/details/multi/advance/AdvancedProcessDefDetailsMultiPresenter.java:62]: that would be in here: https://github.com/kiegroup/appformer/pull/175/files#diff-30dc716694e59aff90189e2110212da5R163

**Review by bsremac — APPROVED** [review]

**Comment by bsremac** [comment]
@cristianonicolai There's no harm in approving certain things twice, even when done by mistake :)

File: jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/requestlist/RequestListPresenter.java [hunk 2]
```diff
@@ -258,28 +253,28 @@ public void callback(Void nothing) {
...
+    @WorkbenchMenu
+    public Menus getMenus() {
         return MenuFactory
-                .newTopLevelMenu(constants.New_Job())
-                .respondsWith(newJobCommand)
-                .endMenu()
-                .newTopLevelCustomMenu(serverTemplateSelectorMenuBuilder)
-                .endMenu()
                 .newTopLevelCustomMenu(new RefreshMenuBuilder(this)).endMenu()
-                .newTopLevelCustomMenu(refreshSelectorMenuBuilder).endMenu()
```
Comment (nmirasch) [inline jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/requestlist/RequestListPresenter.java]: @cristianonicolai I think here we have to keep the refresh option

Comment (nmirasch) [reply jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/requestlist/RequestListPresenter.java]: @cristianonicolai  my mistake, for sure I remove that on my rebase. Sorry

**Pull request merged**